api/src/pcapi/routes/backoffice/offerers/offerer_blueprint.py

```diff
@@ -169,6 +176,11 @@ def _render_offerer_details(offerer_id: int, edit_offerer_form: offerer_forms.Ed
         delete_offerer_form=empty_forms.EmptyForm(),
         show_subscription_tab=show_subscription_tab,
         active_tab=request.args.get("active_tab", "history"),
+        zendesk_sell_synchronisation_form=empty_forms.EmptyForm()
+        if row.has_non_virtual_venues
+        and utils.has_current_user_permission(perm_models.Permissions.MANAGE_PRO_ENTITY)
+        and row.has_non_virtual_venues
```

Comment:
la condition est en double
```suggestion
        if row.has_non_virtual_venues
        and utils.has_current_user_permission(perm_models.Permissions.MANAGE_PRO_ENTITY)
```